dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/report/Entity.java

```diff
@@ -41,10 +41,10 @@
  * @author Morten Olav Hansen <mortenoh@gmail.com>
  */
 @Data
-public class TrackerObjectReport
+public class Entity
```

Comment:
What does `TrackerObjectReport` represent? Where is it used? Just wondering about the name. My understanding so far is that the things that were called `report` are actually `items` on a report to me. So the report aggregates those items.